.github/workflows/pot-file-update.yaml

```diff
@@ -51,4 +51,22 @@ jobs:
           git config user.email github-actions@github.com
           git add .
           git commit -m "update pot-file"
+
+          # do a 5 push retries in case weblate just pushed to the repository
+          # push from weblate could happen easily because webhook updates the translations
+          RETRIES=5
+          DELAY=2
```

Comment:
Why not wait a bit longer and get better chances that the other side is finished? Perhaps 10 seconds? I'd rather hit once successfully than keep banging until it gives :D

Reply:
What is the chance that you will not hit it again after 10 seconds? I understand it's simpler but this is more robust solution from my opinion.

Reply:
Maybe I misunderstood. What is pushing an update to the repo? If it's something else that does not depend on our activity, then you are right and this is ok. If it's a reaction to the other pushes we did in the other matrix jobs, then waiting more would help, but feel free to merge as is.

Reply:
It depends on our activity but it's not executed by us. It works like this:

we push to repo -> GH will execute webhook -> contacting weblate -> weblate process the changes ->  weblate push to our repo

The issue is that we are not able to control or expect availability of anything else than our push. Everything else could be slow based on the systems where it's handled.